deploy/crd.yaml

```diff
@@ -40,6 +40,12 @@ spec:
                 properties:
                   name:
                     type: string
+            ingressAnnotations:
+              description: May be used to configure the ingress objects
+              type: object
+            ingressTLSSecretName:
+              description: May be used to configure the ingress TLS
+              type: string
```

Comment:
Would an `ingressClassName` be useful here too? I've seen that the annotation class name is deprecated now.